src/open_inwoner/scss/components/WarningHeader/WarningHeader.scss

```diff
@@ -0,0 +1,20 @@
+.warning-header {
+  display: flex;
+  box-sizing: border-box;
+  padding: 1rem;
+
+  .warning-header__container {
+    display: flex;
+    align-items: center;
+    padding: 0 4rem;
```

Comment:
'4rem' is a rather odd magic number 🙂, and will behave differently on different types of screens (like retina, and non-retina), so I'd use "padding: 0;" here, and set the max- widths for different screen-widths, because this one should have any padding on mobile anyway. And: we use a '**mobile first**' CSS styling in the entire project so we start with the mobile styling first and then add media-queries for wider screens:
So add something like this and check to see if it looks okay, maybe you need extra styling for screenwidths between 500px & 768px but probably not: 
`.warning-header__container {
    padding: 0; @media (min-width: 768px) {
      max-width: 90%;
    }`